src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts

```diff
@@ -15,7 +15,7 @@ import c from 'highlight.js/lib/languages/c';
 import cpp from 'highlight.js/lib/languages/cpp';
 import csharp from 'highlight.js/lib/languages/csharp';
 import { Log } from 'logging';
-import 'highlight.js/styles/intellij-light.css'
+import { Theme, ThemeInfo } from 'theme';
 
 const { errorLog } = Log.get('CodeBlockMarkupView');
 hljs.registerLanguage('bash', bash);
```

Comment:
I am curious if the code affects startup time?

Reply:
I guess it's not a heavy operation. If we have kind of tracer on client side, we can test it